pkg/cli/cli.go

```diff
@@ -737,6 +737,23 @@ func (self mineCmd) Run() error {
 				profitTracker.Stop()
 			})
 
+			// Reward tracker.
```

Comment:
also always run it in the `dataserver` command and in the `mine` command run only when not connected to a remote db as when connected to a remote DB the remote DB already runs it, look at my new PR to improve this https://github.com/tellor-io/telliot/pull/468

Reply:
Ok, also we need to query these values in the submitter, so we always need a reward instance in the submitter or we can move the query part to a separate package.

Reply:
yes I meant to have an instante in the dataserver command and another instance here

Reply:
this should be in the `if cfg.Db.RemoteHost == "" {` block